Components pages docs offsite overhaul (#633)
mintlify/docs: content/components/responses.mdx

  A response field example
</ResponseField>

```jsx
<ResponseField name="response" type="string" required>
  A response field example
</ResponseField>
```

## Props

<ResponseField name="name" type="string" required>
  The name of the response value.
</ResponseField>

<ResponseField name="type" type="string" required>

>   Expected type of the response value, you can also pass a custom type string

Reviewer: what is this addition for? are you trying to say...

```suggestion
  Expected type of the response value - this can be any arbitrary string.
```

Author: yah much better

<ResponseField name="required" type="boolean">
  Show "required" beside the field name.
</ResponseField>

<ResponseField name="deprecated" type="boolean">

>   Show "deprecated" beside the field name.

Reviewer: ```suggestion
  Whether a field is deprecated or not.
```
